webdriver/tests/bidi/network/fetch_error/fetch_error.py

```diff
@@ -152,19 +152,16 @@ async def test_navigation_id(
     assert fetch_error_event["navigation"] is None
 
     on_fetch_error = wait_for_event(FETCH_ERROR_EVENT)
-    result = await bidi_session.browsing_context.navigate(
+    asyncio.ensure_future(bidi_session.browsing_context.navigate(
```

Comment:
I think we should still await it and get the navigation ID out of it.

Reply:
the navigation command fails without providing navigation ID